arviz/tests/test_plots.py

```diff
@@ -272,15 +272,29 @@ def test_plot_joint_bad(models, model_fit):
     [
         {"plot_kwargs": {"linestyle": "-"}},
         {"contour": True, "fill_last": False},
+        {
+            "contour": True,
+            "contourf_kwargs": {"cmap": "plasma"},

```

Comment:
Why `plasma` and not `viridis`, that is the matplotlib default?

Reply:
To test different values, no particular reason.

Reply:
I have the impression more people like plasma than viridis...